package.json

```diff
@@ -26,7 +26,14 @@
     "url": "https://github.com/meetup/meetup-web-components/issues"
   },
   "homepage": "https://github.com/meetup/meetup-web-components#readme",
-  "dependencies": {
+  "peerDependencies": {
+    "classnames": "^2.2.5",
```

Comment:
See related issue here: https://github.com/meetup/meetup-web-mocks/issues/12

But we shouldn't use `^` notation in peerDependencies since they don't cover major version updates.